packages/vm/src/errors.rs

```diff
@@ -117,7 +117,24 @@ impl From<wasmer_runtime_core::error::ResolveError> for VmError {
 
 impl From<wasmer_runtime_core::error::RuntimeError> for VmError {
     fn from(original: wasmer_runtime_core::error::RuntimeError) -> Self {
-        make_runtime_err(format!("Wasmer runtime error: {:?}", original))
+        use wasmer_runtime_core::error::{InvokeError, RuntimeError};
+
+        fn runtime_error(err: RuntimeError) -> VmError {
+            make_runtime_err(format!("Wasmer runtime error: {:?}", err))
+        }
+
+        match original {
+            // `InvokeError::FailedWithNoError` happens when running out of gas in singlepass v0.17
```

Comment:
Please add a TODO here so we can find this faster to fix it when wasmer fixes it.

But thank you for researching the type and handling it specially.

Reply:
Should i open a tracking issue in this repository?

Reply:
Yes, that is a good idea... Issue to fix it when the wasmer issue is fixed

Reply:
https://github.com/CosmWasm/cosmwasm/issues/375